src/core/declaration.ts

```diff
@@ -114,20 +114,14 @@ export function getDeclaration(ctx: Context, filepath: string, originalImports?:
     directive: stringifyDeclarationImports({ ...originalImports?.directive, ...imports.directive }),
   }
 
-  const head = ctx.options.version === 2.7
```

Comment:
Note that both paths here have returned to the same code, so this check is no longer required.